documentation/source/user_section/installation/mac.rst

```diff
@@ -111,13 +111,13 @@ Procedure:
 
    .. code:: sh
 
-      pip install -e .
+      pip install -e . -f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
Oh, huh! I forgot that "Option 3: Install with pip (experimental)" was a part of our docs, to be completely honest.

Part of me suspects that those instructions don't even work? My hunch is that no one has actually tried them in a long time... But that's probably out of scope for this PR for now, aha.